src/Atata/Attributes/Behaviors/SelectByTextAttribute.cs

```diff
@@ -3,7 +3,7 @@
 namespace Atata
 {
     /// <summary>
-    /// Represents the behavior for option selection of <see cref="Select{T, TOwner}"/> control using option text.
+    /// Represents the behavior for option selection of <see cref="Select{T, TOwner}"/> control using option <c>text</c>.
```

Comment:
Please revert this file. The original is correct:

```suggestion
    /// Represents the behavior for option selection of <see cref="Select{T, TOwner}"/> control using option text.
```